src/system/ui/forms/items/button/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+export const ButtonItem = ( {
+	id,
+	text,
+	btnclass,
+	...rest
+} ) => {
+	return (
+		<button
+
+			key={ id }
+			id={ id }
+			value={text}
+			width="100%"
+			className={"fluid-button "+btnclass}
+			{ ...rest }
+		>{ text }</button>
+	)
+}
```

Comment:
Why not use the FLUID button component here?

Reply:
Okay, Now using fluid button.